### PR TITLE
Update .craft files for 1.14+

### PR DESCRIPTION
**Pull request by HumorousFool**
### Describe in detail what your pull request acomplishes
This pull request updates the craft files to work for 1.14+ and adds new one to fill size gaps.

### Checklist
- [x] Proper internationalization
- [x] Compiled/tested

### CODE REVIEW

File: modules/Movecraft/src/main/resources/types/Airskiff.craft [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    - emerald_block
+    - beacon
+    - flower_pot
+    - "#heads"
+    - "#anvil"
+    - comparator
+    - daylight_detector
+    - redstone_block
+    - hopper
+    - "#rails"
```
Comment (drfiveminusmint) [inline modules/Movecraft/src/main/resources/types/Airskiff.craft:62]: unless I'm very much mistaken this is a behaviour change

File: modules/Movecraft/src/main/resources/types/Airship.craft [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    - nether_brick
+    - enchanting_table
+    - brewing_stand
+    - cauldron
+    - end_stone
+    #- dagon_egg does not work for some reason???
+    - redstone_lamp
+    - ender_chest
+    - tripwire_hook
+    - tripwire
```
Comment (drfiveminusmint) [inline modules/Movecraft/src/main/resources/types/Airship.craft:60]: this is actually a change in behaviour but i don't mind it too much

File: modules/Movecraft/src/main/resources/types/Airship.craft [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+forbiddenBlocks:
+    - piston_head
+
+canCruise: true
+rotateAtMidpoint: true
+canFly: true
+maxHeightLimit: 135
+maxHeightAboveGround: 50
+vertCruiseSkipBlocks: 3
```
Comment (drfiveminusmint) [inline modules/Movecraft/src/main/resources/types/Airship.craft:103]: I understand that this is being done to bring it back in line with Airship Pirates mechanics but I'm unsure if we want the craft files changed like that

File: modules/Movecraft/src/main/resources/types/SubAirship.craft [hunk 4]
```diff
@@ -137,32 +113,31 @@ halfSpeedUnderwater: true
 sinkPercent: 99.0
 overallSinkPercent: 60.0
 moveblocks:
-    152: # redstone represents engines
+    redstone_block: # redstone represents engines
         - 1.0
         - 100.0
-flyblocks:
-    42: # iron represents ballast tanks
+    iron_block: # iron represents ballast tanks
```
Comment (drfiveminusmint) [inline modules/Movecraft/src/main/resources/types/SubAirship.craft:119]: Iron blocks are removed here, again to bring it into line with AP mechanics, but they aren't added to the crafts move-blocks

**Review by drfiveminusmint — COMMENTED** [review]
There are quite a few behaviour changes that I think warrant attention here.

File: modules/Movecraft/src/main/resources/types/Airship.craft [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    - "#trapdoors"
+    - "#stone_bricks"
+    - iron_bars
+    - "#fence_gates"
+    - nether_brick
+    - enchanting_table
+    - brewing_stand
+    - cauldron
+    - end_stone
+    #- dagon_egg does not work for some reason???
```
[the remark this answers is not in the document]
Comment (HumorousFool) [reply modules/Movecraft/src/main/resources/types/Airship.craft:56]: I don't know how I didn't catch that...

File: modules/Movecraft/src/main/resources/types/Airship.craft (hunk 2 above)
Comment (HumorousFool) [reply modules/Movecraft/src/main/resources/types/Airship.craft:60]: I figured having tripwire hooks was useless without the tripwires, I should have mentioned it in the PR description

**Review by HumorousFool — COMMENTED** [review]
Please help I accidentally reviewed this and I don't know how to cancel it

File: modules/Movecraft/src/main/resources/types/SubAirship.craft (hunk 4 above)
Comment (HumorousFool) [reply modules/Movecraft/src/main/resources/types/SubAirship.craft:119]: I don't know why but it is there, you can check it on my actual repo and it is there but it doesn't display here for me either

File: modules/Movecraft/src/main/resources/types/Airskiff.craft (hunk 1 above)
Comment (HumorousFool) [reply modules/Movecraft/src/main/resources/types/Airskiff.craft]: My bad, this was unintended

File: modules/Movecraft/src/main/resources/types/Airship.craft (hunk 3 above)
Comment (HumorousFool) [reply modules/Movecraft/src/main/resources/types/Airship.craft:103]: I also failed to mention that BigAirship CruiseSkipBlocks have also been changed. I suppose it's up to the rest of the team to decide whether these should be changed.

**Comment by HumorousFool** [comment]
The allowedBlocks in Ship and Submarine now pretty much match those in the SubAirship craft file. Theses have sort of been neglected and lack many of the blocks that have been added in recent versions so I thought that this should be part of the update process.

**Pull request merged**